src/Components/Facility/FacilityCreate.tsx

```diff
@@ -158,7 +168,55 @@ export const FacilityCreate = (props: FacilityProps) => {
   const { goBack } = useAppHistory();
   const headerText = !facilityId ? "Create Facility" : "Update Facility";
   const buttonText = !facilityId ? "Save Facility" : "Update Facility";
+  const [imageKey, setImageKey] = useState(Date.now());
+  const [editCoverImage, setEditCoverImage] = useState(false);
+  const authUser = useAuthUser();
+  useMessageListener((data) => console.log(data));
+
+  const {
+    data: facilityData,
+    loading: isLoading,
+    refetch: facilityFetch,
+  } = useQuery(routes.getPermittedFacility, {
+    pathParams: {
+      id: facilityId || "",
+    },
```

Comment:
set prefetch to false if no facility id is present.

Reply:
Respected sir,
I have my mid term exams going on.
As soon as I will get time I will work on the requested changes.